src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,131 @@
+'use strict';
+
+const expensesServices = require('../services/expenses.service');
+const usersServices = require('../services/users.service');
+
+const get = async(req, res) => {
+  const query = req.query;
+  let expenses = [];
+
+  if (!query) {
+    expenses = await expensesServices.getAllExpenses();
```

Comment:
in a real project, you need to handle all possible errors. So wrap in `try catch` all code where an Error can appear